contracts/optimistic-ethereum/OVM/accounts/OVM_ECDSAContractAccount.sol

```diff
@@ -22,8 +22,15 @@ import { Lib_SafeMathWrapper } from "../../libraries/wrappers/Lib_SafeMathWrappe
  */
 contract OVM_ECDSAContractAccount is iOVM_ECDSAContractAccount {
 
+    /*************
+     * Constants *
+     *************/
+
+    // TODO: should be the amount sufficient to cover the gas costs of all of the transactions up
```

Comment:
Is this still a `TODO`?

Reply:
It seems more like a note

Reply:
I agree it's more of a note; we now have a ticket to update intrinsic gas so I think fine to leave as is and update when we do that.